community/cypher/docs/cypher-docs/src/docs/dev/syntax/operators.asciidoc

```diff
@@ -11,6 +11,8 @@ The mathematical operators are `+`, `-`, `*`, `/` and `%`, `^`.
 == Comparison operators ==
 
 The comparison operators are `=`, `<>`, `<`, `>`, `<=`, `>=`, `IS NULL`, and `IS NOT NULL`.
+The operator `LIKE` can be used to compare a string using pattern matching. There are two wildcards
+supported: `%` and `_`. `%` will match zero or more characters and `_` will match exactly one character.
```

Comment:
Also, please add a blank line in front of `The operator LIKE` to make the new sentences a paragraph on their own.